passes/cmds/exec.cc

```diff
@@ -0,0 +1,203 @@
+/*
+ *  yosys -- Yosys Open SYnthesis Suite
+ *
+ *  Copyright (C) 2012  Clifford Wolf <clifford@clifford.at>
```

Comment:
Dunno who that is but I'm pretty sure it's not you :D

Reply:
Sorry, I just copied the header from another existing file.  The email address for this nym is not super stable (or frequently checked), and I am certainly not going to exercise any sort of legal rights, so I hereby assign the copyright for this PR to Claire Wolf.  In the alternative, if a copyright assignment is not legally enforceable in a given jurisdiction, I offer the ISC license as usual.